diff --git a/CMakeLists.txt b/CMakeLists.txt
index e88a604..863ce85 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -101,11 +101,8 @@ endif()
 message(STATUS "LLVM Libraries: ${LLVM_LIBRARIES}")
 
 # Intel XED
-find_package(XED CONFIG REQUIRED)
-add_library(thirdparty_xed INTERFACE)
-target_link_libraries(thirdparty_xed INTERFACE
-  XED::XED
-)
+set(XED_LIBRARY
+  -l libxed)
 
 # Google glog module
 find_package(glog CONFIG REQUIRED)
@@ -235,7 +232,7 @@ target_compile_definitions(remill_settings INTERFACE
   "REMILL_BUILD_SEMANTICS_DIR_SPARC64=\"${REMILL_BUILD_SEMANTICS_DIR_SPARC64}\""
 )
 
-set(THIRDPARTY_LIBRARY_LIST thirdparty_llvm thirdparty_xed thirdparty_glog thirdparty_gflags )
+set(THIRDPARTY_LIBRARY_LIST thirdparty_llvm ${XED_LIBRARY} thirdparty_glog thirdparty_gflags )
 target_link_libraries(remill_settings INTERFACE
   ${THIRDPARTY_LIBRARY_LIST}
 )
